ros2doctor/test/test_qos_compatibility.py

```diff
@@ -122,6 +122,10 @@ def setUpClass(
             filtered_rmw_implementation=rmw_implementation
         )
 
+        # skip zenoh because of the QoS compatibility
+        if rmw_implementation == 'rmw_zenoh_cpp':
+            raise unittest.SkipTest()
```

Comment:
Instead of hardcording `rmw_zenoh_cpp` in the test, can we skip the test if [rclpy.qos_check_compatible()](https://github.com/ros2/rclpy/blob/94f42b6b16de00f1b0543a793c6cd614843ea69e/rclpy/rclpy/qos.py#L523) returns incompatible?

Reply:
This is not a node, this is a launch file who launches some nodes with incompatible qos. Unfortunatelly we can use this